src/pages/NewChatPage.js

```diff
@@ -24,18 +24,18 @@ const propTypes = {
     isGroupChat: PropTypes.bool,
 
     /** Beta features list */
-    betas: PropTypes.arrayOf(PropTypes.string).isRequired,
+    betas: PropTypes.arrayOf(PropTypes.string),
 
     /** All of the personal details for everyone */
-    personalDetails: personalDetailsPropType.isRequired,
+    personalDetails: personalDetailsPropType,
 
     /** All reports shared with the user */
-    reports: PropTypes.objectOf(reportPropTypes).isRequired,
+    reports: PropTypes.objectOf(reportPropTypes),
 
     /** Session of currently logged in user */
     session: PropTypes.shape({
         email: PropTypes.string.isRequired,
-    }).isRequired,
+    }),
```

Comment:
@Prince-Mendiratta  It's weird that the `session` is not detected as unused if it has a `shape` type but detected as unused if `objectOf`.

Reply:
@Prince-Mendiratta Yeah, but it is not part of your lint so let it slide for now.